### DISCUSSION TITLE
Heliview conference green IT

### DISCUSSION THREAD

**Discussion by LadyLowcode** [Green-Software-Foundation/discussions#116] [Events]
Hi all,

Asking for help! https://heliview.com/green-it/ this conference will be in the Netherlands 14th of May. I will not be able to speak there, but they really want someone of the GSF to talk at their conference.
Is anyone available and in the Netherlands at that day? Topic should be GSF related and aimed as the decision-makers in organisations. 
Hope someone can fill in my spot!

In addition they want to ask GSF to help promote the event. Who from GSF can I approach for this?

Best regards,
Marjolein Pordon

**Comment by LadyLowcode** [comment]
@jawache how can I promote my post or tag maybe a group?

**Reply by NAMRATA-WOKE** [reply]
@LadyLowcode thanks for sharing this opportunity! We'll notify a few members in our network and share it in our next newsletter. Can you let me know if there is a deadline you're working against?

**Reply by LadyLowcode** [reply]
By end of next week would be great! Thank you @NAMRATA-WOKE for responding and helping spread the opportunity!

**Comment by muemich** [comment]
@NAMRATA-WOKE our @pinireznik could do it. He is based in Amsterdam
